tests/filesTest.js

```diff
@@ -1242,16 +1361,22 @@ describe('Main: Currently testing files management,', function () {
       ]
 
       const provider = createProvider(true, true)
```

Comment:
should pass now in oc10, can you set it to `const provider = createProvider(false, true)`